packages/sessions-sdk-ts/src/index.ts

```diff
@@ -400,6 +400,15 @@ const getTokenInfo = async (
 
 type TokenInfo = Awaited<ReturnType<typeof getTokenInfo>>[number];
 
+const serializeU16LE = (value: number) => {
+  return Uint8Array.from([
+    // eslint-disable-next-line unicorn/number-literal-case
+    value & 0xff,
```

Comment:
maybe i'm just bad at bit shifts and operations, but i think you should comment out what's happening here for clarity

Reply:
yes please, 100% agree